public/components/layout.js

```diff
@@ -135,7 +145,7 @@ class UserResource extends HTMLElement {
                     + " will note be able to gather additional information about this targeted resource."
                     + " You can supply a different URI or continue with this one.")
                 uriPreview.innerHTML = `<pre>{Not Resolvable}</pre>`
-                localStorage.setItem("userResource", JSON.stringify({"@id":target})) 
+                localStorage.setItem("userResource", JSON.stringify({"@id":target, "creator":objCreator.value}))
```

Comment:
Do not add the `creator` key if the value is an empty string, `undefined`, or `null`